libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/LocaleEx.kt

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
This is not needed anymore right? As `LocaleEx` is an `object`. Is that correct?

Reply:
It's needed for the getLocaleDirectionsRoute function. That function ins't an extension function and it needs to get called from java.